spec/features/copying_files_spec.rb

```diff
@@ -19,7 +19,11 @@
     copy_attributes = copy.file.attributes
     copy_attributes.reject! { |k,v| k == :last_modified }
 
+    copy_acl_grants = copy.file.file.acl.grants
+    original_acl_grants = original.file.file.acl.grants
+
     expect(copy_attributes).to eq(original_attributes)
+    expect(copy_acl_grants).to eq(original_acl_grants)
```

Comment:
Yes, it did fail but you can't see this on Travis because S3 specs are ignored when the ENV vars are not present.

Let me show you the failure for proof: 

```
Run options: include {:focus=>true}

All examples were filtered out; ignoring {:focus=>true}

Randomized with seed 37055
......................................F

Failures:

  1) Copying Files copies an existing file to the specified path
     Failure/Error: expect(copy_acl_grants).to eq(original_acl_grants)

       expected: [#<struct Aws::S3::Types::Grant grantee=#<struct Aws::S3::Types::Grantee display_name="vendor+aws", email_address=nil, id="45836e7b6274ab9b2adab5c56dc23bc4eca041d5cb8c03506f2a2693c96abc59", type="CanonicalUser", uri=nil>, permission="FULL_CONTROL">, #<struct Aws::S3::Types::Grant grantee=#<struct Aws::S3::Types::Grantee display_name=nil, email_address=nil, id=nil, type="Group", uri="http://acs.amazonaws.com/groups/global/AllUsers">, permission="READ">]
            got: [#<struct Aws::S3::Types::Grant grantee=#<struct Aws::S3::Types::Grantee display_name="vendor+aws", email_address=nil, id="45836e7b6274ab9b2adab5c56dc23bc4eca041d5cb8c03506f2a2693c96abc59", type="CanonicalUser", uri=nil>, permission="FULL_CONTROL">]

       (compared using ==)

       Diff:
       @@ -1,3 +1,2 @@
       -["#<struct Aws::S3::Types::Grant grantee=#<struct Aws::S3::Types::Grantee display_name=\"vendor+aws\", email_address=nil, id=\"45836e7b6274ab9b2adab5c56dc23bc4eca041d5cb8c03506f2a2693c96abc59\", type=\"CanonicalUser\", uri=nil>, permission=\"FULL_CONTROL\">",
       - "#<struct Aws::S3::Types::Grant grantee=#<struct Aws::S3::Types::Grantee display_name=nil, email_address=nil, id=nil, type=\"Group\", uri=\"http://acs.amazonaws.com/groups/global/AllUsers\">, permission=\"READ\">"]
       +["#<struct Aws::S3::Types::Grant grantee=#<struct Aws::S3::Types::Grantee display_name=\"vendor+aws\", email_address=nil, id=\"45836e7b6274ab9b2adab5c56dc23bc4eca041d5cb8c03506f2a2693c96abc59\", type=\"CanonicalUser\", uri=nil>, permission=\"FULL_CONTROL\">"]

     # ./spec/features/copying_files_spec.rb:26:in `block (2 levels) in <top (required)>'

Finished in 3.85 seconds (files took 0.48477 seconds to load)
39 examples, 1 failure

Failed examples:

rspec ./spec/features/copying_files_spec.rb:7 # Copying Files copies an existing file to the specified path

Randomized with seed 37055
```